docs/references/database/list.md

```diff
@@ -0,0 +1 @@
+Get a list of all the user's databases. You can use the query params to filter your results. On admin mode, this endpoint will return a list of all of the project's databases. [Learn more about different API modes](/docs/admin).
```

Comment:
We don't have query parameter is only for list documents, we don't yet have it here

Reply:
Good catch. We should fix it here as well: https://appwrite.io/docs/server/functions?sdk=nodejs-default#functionsList